proto/commons/massa/model/v1/execution.proto

```diff
@@ -131,21 +131,26 @@ message AsyncPoolChangeValue {
   // The type of the change
   AsyncPoolChangeType type = 1;
   // AsyncPool message
-  AsyncMessage async_message = 2;
+  oneof message {
+    // Created ledger entry
+    AsyncMessage created_message = 2;
```

Comment:
Out of curiosity, what are those messages?

Reply:
This message could have two different types.